airbyte-integrations/connectors/source-postgres/gradle.properties

```diff
@@ -0,0 +1 @@
+testExecutionConcurrency=-1
```

Comment:
This triggers the junit test execution concurrency (instead of the default gradle concurrency). What this means is that the tests run in different threads rather than different processes.

Reply:
Do we tell also gradle to _not_ do concurrency?

Reply:
Indeed we do, some connectors have `testExecutionConcurrency=1` which effectively disables concurrency.